easybuild/easyconfigs/p/PyGTS/PyGTS-0.3.1-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonPackage'
+
+name = 'PyGTS'
+version = '0.3.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://sourceforge.net/projects/pygts/'
+description = """ PyGTS is a python package used to construct, manipulate, 
+ and perform computations on triangulated surfaces. 
+ It is a hand-crafted and pythonic binding for the GNU Triangulated Surface (GTS) Library. """
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('GLib', '2.47.5'),
+    ('GTS' ,'0.7.6'),
+    ('numpy', '1.8.2', versionsuffix),
```

Comment:
@jrbosch why the old numpy? There's a numpy already included in the Python dependency itself...